test/apparmor.bats

```diff
@@ -19,7 +19,7 @@ function teardown() {
 
     sed -e 's/%VALUE%/,"container\.apparmor\.security\.beta\.kubernetes\.io\/testname1": "runtime\/default"/g' "$TESTDATA"/sandbox_config_seccomp.json > "$TESTDIR"/apparmor1.json
 
-    run crictl runs "$TESTDIR"/apparmor1.json
+    run crictl runp "$TESTDIR"/apparmor1.json
```

Comment:
As of d0fd1f5f, there's still a `runs` over in `test/testdata/README.md` which you probably want to update to `runp`.